File: TersKosegenMatrisi.py
n = int(input("Bir sayı giriniz: "))

for i in range(1, n+1):
    for j in range(n, 0, -1):
        if i == j:
            print(1, end=" ")
        else:
            print(0, end=" ")
    
    print()

File: TersKosegenMatrisi.java
import java.util.Scanner;

public class MainClass {
       
    public static void main(String[] args)
    {
        Scanner scan= new Scanner(System.in);
        
        System.out.println("Bir sayı giriniz: ");
        int num=scan.nextInt();
        
        for(int i=1;i<=num;i++){
            
            for(int j=num;j>=i;j--)
            {
                System.out.print("*");
            }
            System.out.println();
        }
    }
}
